draft-ietf-rtcweb-jsep.xml

```diff
@@ -319,6 +319,19 @@ ipr="trust200902">
         answerer, who first calls setRemoteDescription(sdp [offer]) and
         then later setLocalDescription(sdp [answer]).</t>
 
+        <t>During the offer/answer exchange (i.e., in states other than
+        stable), the outstanding offer is "pending" at the offerer
```

Comment:
Yeah, what do you think I should do for the stable thing? We could shove this text after the state machine...